web/styles/compose.css

```diff
@@ -892,6 +892,7 @@ a.compose_control_button.hide {
 
     &.typeahead {
         background: hsl(0deg 0% 100%);
+        right: 0;
```

Comment:
I think this duplicates what's already set in `search.css`

Reply:
yup! thanks, removed